Documentation/gce-backend.md

```diff
@@ -0,0 +1,72 @@
+# GCE Backend for Flannel
+
```

Comment:
When running on GCE we recommend using the GCE backend which, instead of using encapsulation, manipulates IP routes to achieve maximum performance. Because of this, a separate flannel interface is not created.